lib/module_importer.js

```diff
@@ -33,7 +33,7 @@ function readFirstFile(uri, filenames, cb, examinedFiles) {
   var filename = filenames.shift();
   examinedFiles = examinedFiles || [];
   examinedFiles.push(filename);
-  fs.readFile(filename, function(err, data) {
+  fs.readFile(filename, "utf8", function(err, data) {
```

Comment:
This has the effect of returning a String instead of a Buffer previously.
Was it intended ?